openghg/standardise/_standardise.py

```diff
@@ -207,46 +208,44 @@ def standardise_column(
-        species: Species name or synonym e.g. "ch4"
-        platform: Type of platform. Should be one of:
+        filepath: Path to the input observation file (netCDF).
+        species: Species name or synonym (e.g., "ch4").
+        platform: Type of platform (default is "satellite"). Can be one of:
             - "satellite"
             - "site"
-        satellite: Name of satellite (if relevant). Should include satellite OR site.
-        domain: For satellite only. If data has been selected on an area include the
-            identifier name for domain covered. This can map to previously defined domains
-            (see openghg_defs "domain_info.json" file) or a newly defined domain.
-        selection: For satellite only, identifier for any data selection which has been
-            performed on satellite data. This can be based on any form of filtering, binning etc.
-            but should be unique compared to other selections made e.g. "land", "glint", "upperlimit".
-            If not specified, domain will be used.
-        site : Site code/name (if relevant). Should include satellite OR site.
-        instrument: Instrument name e.g. "TANSO-FTS"
-        network: Name of in-situ or satellite network e.g. "TCCON", "GOSAT"        source_format : Type of data being input e.g. openghg (internal format)
-        store: Name of store to write to
-        if_exists: What to do if existing data is present.
-            - "auto" - checks new and current data for timeseries overlap
-                - adds data if no overlap
-                - raises DataOverlapError if there is an overlap
-            - "new" - just include new data and ignore previous
-            - "combine" - replace and insert new data into current timeseries
-        save_current: Whether to save data in current form and create a new version.
-             - "auto" - this will depend on if_exists input ("auto" -> False), (other -> True)
-             - "y" / "yes" - Save current data exactly as it exists as a separate (previous) version
-             - "n" / "no" - Allow current data to updated / deleted
-        overwrite: Deprecated. This will use options for if_exists="new".
-        force: Force adding of data even if this is identical to data stored.
-        compression: Enable compression in the store
-        compressor: A custom compressor to use. If None, this will default to
-            `Blosc(cname="zstd", clevel=5, shuffle=Blosc.SHUFFLE)`.
-            See https://zarr.readthedocs.io/en/stable/api/codecs.html for more information on compressors.
-        filters: Filters to apply to the data on storage, this defaults to no filtering. See
-            https://zarr.readthedocs.io/en/stable/tutorial.html#filters for more information on picking filters.
-        chunks: Chunking schema to use when storing data. It expects a dictionary of dimension name and chunk size,
+        region: The geographic region covered by the data ("BRAZIL", "INDIA", "UK").
```

Comment:
Looking at this again, perhaps we should update `region` to something like `obs_region` as perhaps this may look too similar to `domain` otherwise.

The alternative would be to use `domain` to describe both but that may get a bit confusing as the `domain` for the associated footprint would then not match the `domain` idea for the satellite data. The purpose of including an area extent keyword (like `obs_region` or `domain`) is to describe the region over which the data has been selected, rather than describing a 2D grid as we have for footprint, flux data etc.